samples/scala-customer-registry-quickstart/README.md

```diff
@@ -0,0 +1,89 @@
+# Quickstart project: Customer Registry
+
+# TODO update for quickstart instructions
```

Comment:
still something TODO?

Reply:
Copied TODO from the Java README, not quite sure what it is about, but if it is needed there it probably is needed here?